WindowsFormsApp10/Form1.cs

```diff
@@ -29,13 +29,13 @@ private DataTable GetDataTable()
         {
             DataTable dt = new DataTable();
 
-            DataColumn dc = new DataColumn("Differencia", typeof(double));
+            DataColumn dc = new DataColumn("Differencia", typeof(string));
```

Comment:
this is a solution, but this is a sample from a real case where the user has a big table and he can't change the type of his column with that data.
I recommend thinking about why this is so, what is the issue with the double type, and whether it is possible to solve the issue at the level of generate numbers.

Reply:
In my home opinion, the real reason of this bug is a floating point precision problem, so we can close this issue by changing type to string or decimal (in last commit I realized it). Am I right?

Reply:
I mean, can I change the "double" type to at least "decimal" type?

Reply:
@telesium, the reason is right, but can you provide the solution without changing double type to anything else?

Reply:
The last solution can be provided to customer in real life task.
The original solution doesn't change the type of dDiferencia variable at all, but the provided is also ok.